ui/tests/integration/components/json-editor-test.js

```diff
@@ -78,4 +78,26 @@ module('Integration | Component | json-editor', function (hooks) {
     });
     assert.dom('.CodeMirror-linenumber').doesNotExist('on readOnly does not show line numbers');
   });
+
+  test('it should render example and restore it', async function (assert) {
+    this.value = null;
+    this.example = 'this is a test example';
+
+    await render(hbs`
+      <JsonEditor
+        @value={{this.value}}
+        @example={{this.example}}
+        @mode="ruby"
+        @valueUpdated={{fn (mut this.value)}}
+      />
+    `);
+
+    assert.dom('.CodeMirror-code').hasText(`1${this.example}`, 'Example renders when there is no value');
```

Comment:
is the `1` something codemirror adds?

Reply:
Ya so I've spent more time then I would like to admit trying to extract only the user inputted text from the code editor for a strict comparison but the `textarea` is actually hidden and then it renders the input in divs as individual code lines and then adds the line numbers.